docs/README.md

```diff
@@ -0,0 +1,3 @@
+# Current Oncall 
+
+<iframe src="https://storage.googleapis.com/knative-infra-oncall/oncall.html" width="100%" height="384"></iframe>
```

Comment:
this html tag does seem to work in markdown file

Reply:
It does. Our linter doesn't like it:

```
docs/README.md:1: MD009 Trailing spaces
docs/README.md:3: MD033 Inline HTML

A detailed description of the rules is available at https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md
```

I'll disable the rule for our repo.

Reply:
It still shows raw html in your fork, would it be different in knative?
https://github.com/adrcunha/test-infra/blob/onca/docs/README.md

Reply:
That's expected, as this markdown file is meant to be viewed through GitHub Pages: https://adrcunha.github.io/test-infra/

Reply:
thanks, good to know